components/print/cups/Makefile

```diff
@@ -25,23 +25,27 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		cups
-COMPONENT_VERSION=	1.7.5
-COMPONENT_REVISION=	3
+COMPONENT_VERSION=	2.3.3
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_PROJECT_URL=	http://www.cups.org/
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC)-source.tar.bz2
-COMPONENT_ARCHIVE_HASH=	\
-    sha256:18cb4c6847dbaaaa05c8b35af787f19dd5c7686970b46548e72c711c6f26bd02
-COMPONENT_ARCHIVE_URL=	https://www.cups.org/software/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC)-source.tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:261fd948bce8647b6d5cb2a1784f0c24cc52b5c4e827b71d726020bcc502f3ee
+COMPONENT_ARCHIVE_URL=  https://github.com/apple/cups/releases/download/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
 COMPONENT_BUGDB=	utility/cups
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+include $(WS_MAKE_RULES)/common.mk
+
+BUILD_BITS= 64
 
```

Comment:
Usually we define BUILD_BITS before the first include (before line #25 here)

Reply:
Yep and that seems to have the effect, that all binaries and libraries are placed into dir/amd64 is that intended? I doubt it is for 64bit only

Reply:
BUILD_BITS= 64 means: only 64-bit libraries and binaries. If we also need 32-bit libraries it should be BUILD_BITS=64_and_32 (or 32_and_64).
If we don't want both binaries the typical way to do it is to
PREFERRED= 64
and not include the resulting 32-bit binaries in the manifest (eg. commented out).

Reply:
Well we ony need 64bit. But in the locations correct locations rather than bin/amd64

Reply:
Fixed now via manifests